### ISSUE TITLE
viewpager in EpoxyRecyclerVIew

### ISSUE THREAD

**Issue by marko995** [airbnb/epoxy#607]
Hello,
i need to implement viewpager in epoxyRecyclerView, but problem is time to inflate and bind.
Does it exist any way to inflate (call bindView) when add model to controller ?

**Comment by elihart** [comment]
No, the whole point of Epoxy is that it doesn't create the view when the model is built.

For view paging like behavior nested in a vertical recyclerview I would recommend using a horizontal recyclerview carousel, possibly with custom snapping behavior
